packages/medusa/src/api/routes/admin/customer-groups/add-customers-batch.ts

```diff
@@ -118,6 +105,23 @@ export default async (req: Request, res: Response) => {
   res.status(200).json({ customer_group })
 }
 
+/**
+ * @schema AdminPostCustomerGroupsGroupCustomersBatchReq
+ * type: object
+ * required:
+ *   - customer_ids
+ * properties:
+ *   customer_ids:
+ *     description: "The ids of the customers to add"
+ *     type: array
+ *     items:
+ *       required:
+ *         - id
+ *       properties:
```

Comment:
I remember you created a previous PR regarding adding explicit `type: object`. Should we add it here as well?